app/globals.css

```diff
@@ -24,18 +24,9 @@
     }
 
     body {
-        @apply grid;
-        @apply bg-neutral-50;
-        grid-template-columns:
-      1fr
-      min(1000px, 100%)
-      1fr;
+        @apply bg-white;
```

Comment:
what happened to the styles that got removed here? that's intentional?

Reply:
Yes! Made the whole page styling weirdly boxy so I removed